mets.md

```diff
@@ -160,6 +199,83 @@ encodings of the same page.
 </mets:structMap>
 ```
 
+## OCR-D structMap
+
+A METS can contain different `structMap`. These are defined by the `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL`>) described in more detail.	
+A document structure determined by OCR-D is described by `TYPE="OCR-D-LOGICAL"`.
+
+attributes in `structMap` | description
+--                        | --
+`LABEL`                   | contains the character recognition result  e.g. the title of the chapter
```

Comment:
```suggestion
`LABEL`                   | contains the recognized text of a structuring element, e.g. the title of the chapter
```

Reply:
->structuring element : a title of the chapter is not an element, that´s only text.
better: `contains the recognized text e.g. the title of the chapter`

Reply:
See discussion below.

```suggestion
`LABEL` | contains the recognized text of a structuring component, e.g. the title of a chapter
```